import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java

```diff
@@ -92,6 +92,32 @@
   )
   String monitoringProjectId = "";
 
+  @Parameter(
+      names = {"--tag_edit_remove"},
```

Comment:
Nitpick: can we rename these to:
--redact_remove_list
--redact_keep_list
--redact_filter_profile?
I think using the full name gives more details, especially for the filter profile. Using redact as the prefix helps the user understand that these are for the redact library.

Reply:
Ok